### PR TITLE
feat: update sessions

### PR DESCRIPTION
**Pull request by Ko1103**
- CdrTokenのカラム追加
- stringにformat date-timeを追加

### CODE REVIEW

**Review by ThaddeusJiang — COMMENTED** [review]
@Ko1103 update session の Responses がまだ定義されてない、一緒に定義を作成しますか？

`200, 400, 401, 404` など

CleanShot 2023-08-07 at 13 51 39

**Review by seiya0429 — APPROVED** [review]
LGTM

File: openapi/paths/sessions.yaml [hunk 1]
```diff
@@ -15,10 +15,17 @@ get:
   responses:
     "200":
       description: Success
+      headers:
+        Link:
+          $ref: ../components/headers/Link.yaml
+        X-Total-Count:
+          $ref: ../components/headers/X-Total-Count.yaml
+        X-Limit:
+          $ref: ../components/headers/X-Limit.yaml
```
Comment (Ko1103) [inline openapi/paths/sessions.yaml:24]: All request with pagenation need these headers.

File: openapi/paths/sessions.yaml [hunk 2]
```diff
@@ -15,10 +15,17 @@ get:
...
       content:
         application/json:
           schema:
             $ref: ../components/schemas/Sessions.yaml
-    "403":
-      description: Forbidden
-      $ref: ../components/responses/Problem.yaml
+    "401":
+      description: Unauthorized
+      $ref: ../components/responses/Unauthorized.yaml
```
Comment (Ko1103) [inline openapi/paths/sessions.yaml:31]: I think this request only has 401, 403 will be not respond.

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ info:
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   x-logo:
     url: https://redocly.github.io/openapi-template/logo.png
-  description: >
+  description: OCPI 2.2.1 speficication
```
Comment (Ko1103) [inline openapi/openapi.yaml:14]: need something string

File: .redocly.yaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # See https://redoc.ly/docs/cli/configuration/ for more information.
-apiDefinitions:
-  main: openapi/openapi.yaml
+apis:
+  main:
+    root: .openapi/openapi.yaml
```
Comment (Ko1103) [inline .redocly.yaml:4]: Means same but update to new style.

File: openapi/components/headers/Link.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+schema:
```
Comment (Ko1103) [inline openapi/components/headers/Link.yaml:1]: I used [this](https://github.com/Redocly/api-definitions/blob/master/openapi/paths/3dsecure.yaml) as a reference

**Review by ThaddeusJiang — APPROVED** [review]
LGTM

**Pull request merged**